functions/expire-prepared-check/v1.js

```diff
@@ -51,6 +51,9 @@ async function deleteExpiredChecks (logger) {
     logger.error(`expire-prepared-checks: ERROR: deleteExpiredChecks(): ${error.message}`)
   }
 
+  const checkCodes = entities.map(e => e.checkCode.toUpperCase())
+  logger(`expire-prepared-checks: Removed ${entities.length} checks from \`${preparedCheckTable}\`:\n${checkCodes.join('\n')}`)
```

Comment:
This is a list of 500 checkCodes.  That's probably too much to log.

Reply:
@danwilliams per Jons comment, can we snip this down to a max of 10 please?